### DISCUSSION TITLE
docs/integration/java/jenkins%20plugin

### DISCUSSION THREAD

**Discussion by giscus** [casdoor/casdoor#1402] [Docs comments]
# docs/integration/java/jenkins%20plugin

Using Casdoor plugin for your Jenkins security

https://casdoor.org/docs/integration/java/jenkins%20plugin

**Comment by bluicezhen** [comment]
no jar file download link

**Comment by hsluoyz** [comment]
Issue created here: https://github.com/casdoor/casdoor/issues/1401
